adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/iOS/AuthenticationContinuationHelper.cs

```diff
@@ -41,7 +41,6 @@ public static class AuthenticationContinuationHelper
         /// Returns if the response is from the broker app
         /// </summary>
         /// <param name="sourceApplication">application bundle id</param>
-        /// <returns></returns>
```

Comment:
>/// <returns></returns> [](start = 8, length = 23)

should probably have a meaningful text instead. 
E.g.
True if the response is from broker, False otherwise.

Reply:
Assuming this is included in the dll which remains

---
In reply to: [212023486](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1220#discussion_r212023486) [](ancestors = 212023486)